From 3307c8bf8ec5b8ae117f4fe9faf40c09f08edfb5 Mon Sep 17 00:00:00 2001
From: Aelin <aelin@postmarketos.org>
Date: Sat, 12 Sep 2026 13:40:55 +0200
Subject: [PATCH 07/10] Allow systemd-timesyncd to read network sysctls

systemd-timesyncd reads the value of
/proc/sys/net/ipv6/conf/all/disable_ipv6, which is currently not
allowed.
---
 policy/modules/system/systemd.te | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/policy/modules/system/systemd.te b/policy/modules/system/systemd.te
index f38ff3a0e..5e97bc2fe 100644
--- a/policy/modules/system/systemd.te
+++ b/policy/modules/system/systemd.te
@@ -1228,7 +1228,9 @@ allow systemd_timedated_t systemd_networkd_var_run_t:dir watch_dir_perms;
 list_dirs_pattern(systemd_timedated_t, systemd_networkd_var_run_t, systemd_networkd_var_run_t)
 read_files_pattern(systemd_timedated_t, systemd_networkd_var_run_t, systemd_networkd_var_run_t)
 
+kernel_read_net_sysctls(systemd_timedated_t)
 kernel_rw_psi(systemd_timedated_t)
+kernel_search_network_sysctl(systemd_timedated_t)
 
 corecmd_exec_bin(systemd_timedated_t)
 corecmd_exec_shell(systemd_timedated_t)
-- 
2.55.0

